includes/core.php

```diff
@@ -57,50 +58,29 @@ function get_current_draft( $project_id, $document_id ) {
 }
 
 /**
- * Verify whether or not the current environment meets plugin requirements.
- *
- * The requirements for the plugin should be documented in the plugin README, but include:
- *
- * - PHP >= 5.3        - Namespace support, though the plugin will fail before reaching this check
- *                       if namespaces are unsupported.
- * - dom extension     - Used by DOMDocument in formatters.php.
- * - mcrypt extension  - Used as a backup for older systems that don't support PHP 7's random_bytes().
- * - openssl extension - Used to securely encrypt Airstory credentials.
- *
- * @return bool True if all requirements are met, false otherwise.
- */
-function check_requirements() {
-	$requirements_met = true;
-
-	// Find any missing extensions; $missing_exts will contain any that fail extension_loaded().
-	$extensions = array( 'dom', 'mcrypt', 'openssl' );
-	if ( array_filter( $extensions, 'extension_loaded' ) !== $extensions ) {
-		$requirements_met = false;
-	}
-
-	return $requirements_met;
```

Comment:
That's in the deleted code, but the original reasoning was because we were checking for PHP extensions in a loop; if we returned immediately after one true/false value was found, we'd be missing a big chunk of our checks.

That being said, when building it with simple booleans, you're absolutely correct: returning true at the end or false anytime we hit a missing dependency would have worked fine.